oak_functions/client/java/src/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -178,13 +201,13 @@ boolean verifyServerIdentity(byte[] serializedServerIdentity,
     if (!verifier.verify()) {
       logger.log(Level.WARNING, "Verification of the ServerIdentity failed.");
       return false;
-    };
+    }
 
     return true;
   }
 
   @Override
-  protected void finalize() throws Throwable {
+  public void finalize() {
```

Comment:
Why has this become public?

Reply:
Needs to be explicitly called so that Oak Functions server doesn't have to wait for Java garbage collector to stop a gRPC session.
Added a comment

Reply:
I think the preferred approach is to have a separate public method (e.g. `stop()` or `close()`) to do that and then call stop from the protected `finalize()`.